modules/040-node-manager/hooks/handle_draining.go

```diff
@@ -54,11 +55,8 @@ var _ = sdk.RegisterFunc(&go_hook.HookConfig{
 			FilterFunc: drainFilter,
 		},
 	},
-	Schedule: []go_hook.ScheduleConfig{
-		{
-			Name:    "draining_schedule",
-			Crontab: "* * * * *",
-		},
+	Settings: &go_hook.HookConfigSettings{
+		ExecutionMinInterval: 30 * time.Second,
```

Comment:
What if we switch to labels instead of annotations to have more reactive UX?

Reply:
Well, it would be more reactive but these kinds of labels look weird.
Annotations is a kind of status of the object, some realtime information
while label is an attribute, which describe node in some way